### ISSUE TITLE
Add autoremove option to apt module

### ISSUE THREAD

**Issue by parberge** [ansible/ansible-modules-core#965]
##### Issue Type:

Feature Idea
##### Summary:

I'd like "autoremove" option added to the apt module.

**Comment by alvarezsantiago** [comment]
+1

**Comment by tersmitten** [comment]
Duplicates #1976

**Comment by tersmitten** [comment]
I'm working on more states: https://github.com/tersmitten/ansible-modules-core/compare/support-apt-get-autoclean

I'll try to implement this one too.

**Comment by marccarre** [comment]
+1 

I'd like to be able to run `apt-get -y autoremove --purge <package name>`.
I guess if/once the `autoremove` option is added, it would look like:

```
apt: name=<package name> force=yes autoremove=yes purge=yes state=absent
```

**Comment by kfries** [comment]
If you are using **autoremove** why would you be specifying a name parameter?  Autoremove should remove all unneeded packages.  Shouldn't the command look like:

```
apt: autoremove=yes purge=yes
```

**Comment by sivel** [comment]
This has recently been added to the apt module in #2974 and will be included in Ansible 2.1.
